src/structures/MessageEmbed.js

```diff
@@ -61,6 +73,12 @@ class MessageEmbed {
      * @type {MessageEmbedProvider}
      */
     this.provider = data.provider ? new MessageEmbedProvider(this, data.provider) : null;
+
+    /**
+     * The footer of this embed
+     * @type {MessageEmbedFooter}
+     */
+    this.footer = new MessageEmbedFooter(data.footer);
```

Comment:
Should be `this.footer = data.footer ? new MessageEmbedFooter(this, data.footer) : null;`